web/src/pm_list.ts

```diff
@@ -39,8 +39,10 @@ export function close(): void {
 }
 
 export function _build_direct_messages_list(): vdom.Tag<PMNode> {
-    const conversations = pm_list_data.get_conversations();
-    const pm_list_info = pm_list_data.get_list_info(zoomed);
+    const $filter = $(".direct-messages-list-filter").expectOne();
+    const search_term = ($filter.val() ?? "").toString();
```

Comment:
Is this `toString` necessary? I think it might be a workaround for not specifying the type of `$filter` correctly, as `$<HTMLInputElement>("input.direct-messages-list-filter")` so that it knows we're getting the value of something whose `.val()` type is more constrained.

See this example:
`const new_keyword = $<HTMLInputElement>("input#inbox-search").val() ?? "";`

Reply:
Fixed!